reliability/single-fit/trace/src/test/kotlin/com/datadog/android/trace/integration/assertj/PayloadAssert.kt

```diff
@@ -0,0 +1,266 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.trace.integration.assertj
```

Comment:
> [!NOTE]
> Based on our testing guidelines, this should be in a package named `com.datadog.android.trace.integration.tests.assertj`

Reply:
nope, it is in the right package there, `assertj` is not part of tests...maybe I should put it there though 🤔